actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -799,6 +805,12 @@ def init_with(coder) # :nodoc:
       end
     end
 
+    def each_nested_attribute # :nodoc:
```

Comment:
It looks like this is still public. Making it `protected` should still work, and remove the need to use `:nodoc:`.